### PR TITLE
build: release v1.0.0

### PR DESCRIPTION
**Pull request by istankovic**
Contains dependency updates (please see individual commits) plus the release-related material.

----
##### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: crypto/Cargo.toml [hunk 1]
```diff
@@ -102,7 +102,7 @@ fluvio-wasm-timer = "0.2"
 time = { version = "0.3", features = ["wasm-bindgen"] }
 
 [dev-dependencies.core-crypto-keystore]
-version = "^1.0.0-rc.60"
+version = "^1.0.0"
```
Comment (augustocdias) [inline crypto/Cargo.toml:105]: can you move this to the workspace dependency also?

File: crypto/Cargo.toml [hunk 2]
```diff
@@ -115,7 +115,7 @@ version = "0.5"
 features = ["async_std", "html_reports"]
 
 [dev-dependencies.core-crypto-attributes]
-version = "^1.0.0-rc.60"
+version = "^1.0.0"
```
Comment (augustocdias) [inline crypto/Cargo.toml:118]: can you move this to the workspace dependency also?

**Review by augustocdias — CHANGES_REQUESTED** [review]

File: crypto/Cargo.toml (hunk 2 above)
Comment (istankovic) [reply crypto/Cargo.toml:118]: Sure, good catch!

Comment (istankovic) [reply crypto/Cargo.toml]: Done.

File: crypto/Cargo.toml (hunk 1 above)
Comment (istankovic) [reply crypto/Cargo.toml]: Done.

**Pull request merged**